django_project/feti/templates/feti_base.html

```diff
@@ -70,18 +70,24 @@
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                 </button>
-                <a class="navbar-brand topnav" href="/">FETI</a>
+                <a class="navbar-brand topnav" href="/#">FETI</a>
             </div>
             <!-- Collect the nav links, forms, and other content for toggling -->
             <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                 <ul class="nav navbar-nav navbar-right">
-                    <li>
+                    <li class="dropdown">
                         {% if user.is_authenticated %}
-                            {% if user.is_staff %}
-                                <a href="/feti-admin">Admin Site</a>
-                            {% else %}
-                                <a href="/profile/{{ user.username }}">{{ user.username }}</a>
-                            {% endif %}
+                            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
+                                {{ user.username }}
+                                <span class="caret"></span></a>
+                            <ul class="dropdown-menu">
+                                <li><a href="/profile/{{ user.username }}">Your profile</a></li>
```

Comment:
profile page is not in our project now. 
but the url is already there

of /profile page, i put campus admin
maybe i will update this